main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <iomanip>
+
+// Assignment one stock pricing and profit
+
+int main() {
+    // Declare Constants
+    int SHARES_BOUGHT = 1000;
+    int SHARES_SOLD = 1000;
+    double COMMISSION_RATE = 0.02;
+    double PURCHASE_PRICE =45.50;
+    double SALE_PRICE = 56.90;
+
+    // Calculate the cost of the stock
+    double purchase_cost_of_stock = SHARES_BOUGHT * PURCHASE_PRICE;
+    // Calculate the selling price of the stock
+    double sell_price_of_stock = SHARES_SOLD * SALE_PRICE;
+    // Calculate the purchasing commission
+    double purchase_commission = purchase_cost_of_stock * COMMISSION_RATE;
+    // Calculate the selling commission
+    double sell_commission = sell_price_of_stock * COMMISSION_RATE;
+    // Calculate the profit/loss
+    double profit = (sell_price_of_stock - purchase_cost_of_stock) - (sell_commission + purchase_commission);
+
+    // Output the following
+    // The amount of money Joe paid for the stock.
+    // The amount of commission Joe paid his broker when he bought the stock.
+    // The amount that Joe sold the stock for.
+    // The amount of commission Joe paid his broker when he sold the stock.
+    // Display the amount of profit that Joe made after selling his stock and
+    // paying the two commissions to his broker. (If the amount of profit that
+    // your program displays is a negative number, then Joe lost money on the transaction.)
```

Comment:
While I like your verbosity these are likely more comments than we need. What would be better is if we considered more declarative is if you paired the variable representing the calculation with its output with a single comment of its intent like this
```
// Purchasing Stocks
double purchase_cost_of_stock = SHARES_BOUGHT * PURCHASE_PRICE;
double purchase_commission = purchase_cost_of_stock * COMMISSION_RATE;
std::cout << "Original stock purchase price: $" << purchase_cost_of_stock << std::endl;
std::cout << "Original stock purchase commission: $" << purchase_commission<< std::endl;

// Sales of Stock
double sell_price_of_stock = SHARES_SOLD * SALE_PRICE;
double sell_commission = sell_price_of_stock * COMMISSION_RATE;
std::cout << "Original stock sell price: $" << sell_price_of_stock << std::endl;
std::cout << "Original stock sell commission: $" << sell_commission << std::endl;
```
This bakes a lot of meaning from the actual lines of code by having them organized into logic blocks separated by whitespace as opposed to lots of text.